### PR TITLE
docs: update UG and PPP

### PR DESCRIPTION
**Pull request by Wingedevil**

### CODE REVIEW

**Comment by SQwQ** [comment]
Please resolve branch conflicts

File: src/test/java/seedu/address/logic/commands/AliasCommandIntegrationTest.java [hunk 1]
```diff
@@ -22,20 +22,39 @@ public void setUp() {
...
+        String alias1 = "TEST ALIAS";
+        String aliasTo1 = "help";
+        String aliasUsage1 = "TEST ALIAS";
+        String aliasUsageExpected1 = "help";
+        String alias2 = "TEST ADD";
+        String aliasTo2 = "add";
+        String aliasUsage2 = "TEST ADD n/John Smith p/91234567 e/john@smith.co.uk "
+                + "a/12 Downing St Westminster, London SW1A 2AD, UK";
+        String aliasUsageExpected2 = "add n/John Smith p/91234567 e/john@smith.co.uk "
+                + "a/12 Downing St Westminster, London SW1A 2AD, UK";
```
Comment (Q-gabe) [inline src/test/java/seedu/address/logic/commands/AliasCommandIntegrationTest.java]: Test Aliases could perhaps be set as a static variable in a separate TestUtil class actually. Something like TypicalAliases() could be useful for future extension of the alias feature.

File: src/test/java/seedu/address/logic/commands/AliasCommandIntegrationTest.java [hunk 2]
```diff
@@ -22,20 +22,39 @@ public void setUp() {
...
+        assertTrue(model.applyAlias(aliasUsage2).equals(aliasUsageExpected2));
+
+        expectedModel.removeAlias(alias1.toLowerCase());
+        assertCommandSuccess(new UnaliasCommand(alias1), model,
+                String.format(UnaliasCommand.MESSAGE_SUCCESS, alias1.toLowerCase()), expectedModel);
+
+        expectedModel.removeAlias(alias2.toLowerCase());
+        assertCommandSuccess(new UnaliasCommand(alias2), model,
+                String.format(UnaliasCommand.MESSAGE_SUCCESS, alias2.toLowerCase()), expectedModel);
     }
```
Comment (Q-gabe) [inline src/test/java/seedu/address/logic/commands/AliasCommandIntegrationTest.java]: Comments for each equivalence partition would be a nice addition for each individual assertCommandSuccess() call.

File: src/main/java/seedu/address/ui/AliasListWindow.java [hunk 3]
```diff
@@ -54,6 +58,7 @@ public void setup(String aliases) {
      * </ul>
      */
     public void show() {
+        logger.fine("Showing list of aliases.");
```
Comment (Q-gabe) [inline src/main/java/seedu/address/ui/AliasListWindow.java]: Use of logger is nice!

**Pull request merged**